.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,5 +20,6 @@ about: Tell us about a problem you are experiencing
 **Environment:**
 
 - Cluster-api version:
+- Minikube version:
```

Comment:
Since there are two supported bootstrappers, I would suggest this be changed to:
```suggestion
- Minikube/KIND version:
```

Reply:
Done